csharp/src/Microsoft.ML.OnnxRuntime/Training/TrainingSession.shared.cs

```diff
@@ -9,12 +9,29 @@
 namespace Microsoft.ML.OnnxRuntime
 {
 #if __ENABLE_TRAINING_APIS__
+    /// <summary>
+    /// This class defines utility methods for training.
+    /// </summary>
+    public class TrainingUtils
```

Comment:
should this be: :IDisposable?

Reply:
All methods in this class are static methods. Not sure if it would still require IDisposable inheritance?